brewchat/src/main/java/com/example/brewchat/Activities/ChatManagerActivity.java

```diff
@@ -0,0 +1,76 @@
+package com.example.brewchat.Activities;
+
+import android.net.Uri;
+import android.support.v7.app.ActionBarActivity;
+import android.support.v7.app.ActionBar;
+import android.support.v4.app.Fragment;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.os.Build;
+
+import com.example.brewchat.Fragments.ChatManagerFragment;
+import com.example.brewchat.R;
+
+public class ChatManagerActivity extends ActionBarActivity implements ChatManagerFragment.OnFragmentInteractionListener {
```

Comment:
Please note that `ActionBarActivity` has been deprecated in favor of `AppCompatActivity`.

See https://developer.android.com/reference/android/support/v7/app/ActionBarActivity.html
And http://developer.android.com/tools/support-library/index.html